tools/test-js-with-node

```diff
@@ -102,6 +103,12 @@ parser.add_argument('--force', dest='force',
                     action="store_true",
                     default=False, help='Run tests despite possible problems.')
 parser.add_argument('args', nargs=argparse.REMAINDER)
+parser.add_argument('--parallel', dest='parallel',
+                    action='store',
+                    type=int,
+                    default=4,
```

Comment:
Should this default to `os.cpu_count()` (like `test_backend`)?

Reply:
It should be, yes.  Discussing with @adnrs96, that improved performance a bit on his dual core.  That would also require a change in `/tools/ci/frontend` similar to 7c94d350d40d17e6b85d27a4cb024f90607aef0a, correct?